example/src/lib.rs

```diff
@@ -11,54 +11,73 @@ mod ffi {
     use writeable::Writeable;
 
     #[diplomat::opaque]
+    /// A decimal number. See [the Rust docs](https://unicode-org.github.io/icu4x-docs/doc/fixed_decimal/decimal/struct.FixedDecimal.html) for more information.
     pub struct ICU4XFixedDecimal(pub FixedDecimal);
 
     impl ICU4XFixedDecimal {
+        /// Construct an [`ICU4XFixedDecimal`] from an integer.
         fn new(v: i32) -> Box<ICU4XFixedDecimal> {
             Box::new(ICU4XFixedDecimal(FixedDecimal::from(v)))
         }
 
+        /// Multiply the [`ICU4XFixedDecimal`] by a given power of ten.
+        /// See [the Rust docs](https://unicode-org.github.io/icu4x-docs/doc/fixed_decimal/decimal/struct.FixedDecimal.html#method.multiply_pow10) for more information.
         fn multiply_pow10(&mut self, power: i16) {
             self.0.multiply_pow10(power).unwrap();
         }
```

Comment:
Request: Try to make the source text more concise; maybe something like

```rust
        /// Multiply `self` by a given power of ten.
        #[diplomat::rustlink(fixed_decimal::FixedDecimal::multiply_pow10)]
        fn multiply_pow10(&mut self, power: i16) {
            self.0.multiply_pow10(power).unwrap();
        }
```

which could then output to something like (not sure of the RST syntax)

```rst
    .. js:function:: multiply_pow10

        Multiply `self` by a given power of ten.

        Details: `Rust docs <https://unicode-org.github.io/icu4x-docs/doc/fixed_decimal/decimal/struct.FixedDecimal.html#method.multiply_pow10>`
        Enclosing class: [``ICU4XFixedDecimal``]
```

Reply:
You'd also need to say that it's a struct on the rust side, but yes.

`rustlink(struct, fixed_decimal::...)`

We can just support structs for now

Reply:
Filed a followup #21